app/client/src/modules/ui-builder/ui/wds/WDSCustomWidget/component/appsmithConsole.js

```
@@ -0,0 +1,35 @@
+      },
+      "*",
+    );
+  };
+
+  const createProxy = (method) =>
+    new Proxy(nativeConsole[method], {
+      apply(target, _this, args) {
+        try {
+          postMessage(method, args);
+        } finally {
+          return Reflect.apply(target, _this, args);
```

Biome (1.9.4)

[error] Unsafe usage of 'return'.

'return' in 'finally' overwrites the control flow statements inside 'try' and 'catch'.

(lint/correctness/noUnsafeFinally)